patches/chrome-android-features-vr-java-src-org-chromium-chrome-browser-vr-VrFallbackUtils.java.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/android/features/vr/java/src/org/chromium/chrome/browser/vr/VrFallbackUtils.java b/chrome/android/features/vr/java/src/org/chromium/chrome/browser/vr/VrFallbackUtils.java
+index ef8a7d1f59b21c6c65f41009054f959a90301360..563ef806e93ee94dd39df1705b786c00ae619f46 100644
+--- a/chrome/android/features/vr/java/src/org/chromium/chrome/browser/vr/VrFallbackUtils.java
++++ b/chrome/android/features/vr/java/src/org/chromium/chrome/browser/vr/VrFallbackUtils.java
+@@ -23,6 +23,7 @@ import org.chromium.content_public.browser.UiThreadTaskTraits;
+     private static final int PREPARING_VR_NOTIFICATION_TIMEOUT_MS = 5000;
+     private static final int PREPARING_VR_NOTIFICATION_DELAY_MS = 2000;
+ 
++    @android.annotation.SuppressLint("WrongConstant")
```

Comment:
https://github.com/brave/brave-core/pull/12483/commits/da21fe22c0060c77fa3502b5d473047611ae8d47#r823099908

Reply:
yes, they filter it in `chrome/android/expectations/lint-baseline.xml`, but due to added variables there warning text doesn't match what they have in that file